src/Raven.Client/Json/BlittableJsonWriter.cs

```diff
@@ -553,7 +553,16 @@ public override void WriteValue(object value)
             switch (value)
             {
                 case BlittableJsonReaderObject readerObject:
-                    _manualBlittableJsonDocumentBuilder.WriteEmbeddedBlittableDocument(readerObject);
+                    if (false == readerObject.HasParent)
+                    {
+                        _manualBlittableJsonDocumentBuilder.WriteEmbeddedBlittableDocument(readerObject);
+                    }
+                    else
+                    {
+                        throw new NotImplementedException("Can't write value of non root blittable");
+//                        var clonedBlittable = readerObject.Clone(_context);
```

Comment:
Avoid leaving commented code unless it is commented and meant to stay as a warning what _not_ to do.

Reply:
done